src/QueryM/WaitUntilSlot.purs

```diff
@@ -86,12 +84,12 @@ waitUntilSlot futureSlot =
   retryDelay :: Milliseconds
   retryDelay = wrap 1000.0
 
-  logLag :: BigInt -> Milliseconds -> QueryM Unit
+  logLag :: Number -> Milliseconds -> QueryM Unit
```

Comment:
Why not `SlotLength`?

Reply:
I think it would be worse as it only has 3 calls (is defined locally to `waitUntilSlot` ) and is used always with the result of a function that returns a `Number` instead of `SlotLength`  ( the function is `getSlotLength` ) .